src/routes/accommodation.route.js

```diff
@@ -118,4 +118,40 @@ router.get('/:accomodationId', verifyToken.headerToken, verifyUser, Accommodatio
  *            type: string
  */
 router.post('/booking', Validate.bookingValidation(), isValid, verifyToken.headerToken, AccomodationMiddleware.validateDates, AccomodationMiddleware.checkBookingFacilitiesAvailability, Accommodation.bookAccommodation);
+/**
+ * @swagger
+ *
+ * /accommodation/accommodationID:
+ *    patch:
+ *      summary: Like or unlike an accommodation
+ *      tags: [Accommodation]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check authentication
+ *         required: true
+ *         type: string
+ *      responses:
+ *        "200":
+ *          description: Like was successfully processed.
+ */
+router.patch('/:accommodationId', Validate.likeOrUnlikeValidation(), isValid, verifyToken.headerToken, Accommodation.likeOrUnlike);
+/**
+ * @swagger
+ *
+ * /accommodation/accommodationID/user:
+ *    get:
+ *      summary: Check if user has already like or unlike an accommodation
+ *      tags: [Accommodation]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check authentication
+ *         required: true
+ *         type: string
+ *      responses:
+ *        "200":
+ *          description: Check if user has already like or unlike an accommodation.
+ */
+router.get('/:accommodationId/user', isValid, verifyToken.headerToken, Accommodation.hasUserLikeOrUnlike);
```

Comment:
@VedaDom is the user under a particular accommodation? that's what that route says! Kindly come up with another route which is more convenient.